torch/onnx/_internal/exporter.py

```diff
@@ -1002,7 +1002,10 @@ def save(
                     # ignore duplicate
                     continue
                 try:
-                    extra_state_dict = torch.load(path)
+                    # Loads checkpoint using memory-map on CPU to succeed with large models
+                    extra_state_dict = torch.load(
+                        path, map_location="cpu", mmap=True, weights_only=True
```

Comment:
noticed `weights_only=True` is added. is this intended?

Reply:
It is a security feature. when `weights_only=False`, malicious pickled checkpoints can execute code on the machine.
If the checkpoint can be loaded with `weights_only=True`, we should - but I am experimenting with it